src/OpenIddict.Core/IOpenIddictStore.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNet.Identity;
+
+namespace OpenIddict {
+    public interface IOpenIddictStore<TUser, TApplication> : IUserStore<TUser> where TUser : class where TApplication : class {
```

Comment:
Do we need to inherit from IUserStore ? we are not using TUser for anything on the store.